MCprep_addon/spawner/meshswap.py

```diff
@@ -23,11 +23,15 @@
 import time
 
 import bpy
+from bpy.types import (
+  Context, Object, Collection
+)
```

Comment:
Same thing as mentioned earlier
```suggestion
from bpy.types import (
  Context, Collection
)
```